### PR TITLE
Mobile finish setup

### PR DESCRIPTION
**Pull request by Tshepo45**
-Modify index.html styles.css mit.md and README.md files
-Add index.html styles.css mit.md and README.md files

### CODE REVIEW

**Comment by AdnanOlarmmi** [comment]
Hi @Tshepo45 😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
 In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement, and making it professional exactly like that of Microverse.
Kindly complete your readme section, you can follow the readme in [this repo](https://github.com/AdnanOlarmmi/Technopreneur) for inspiration.


Your readme isn't well structured.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html [hunk 1]
```diff
@@ -34,15 +34,7 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
...
-
-
-
-
-
-   
-
-
-    <div class="Container2">
+   <div class="Container2">
```
Comment (V-Blaze) [inline index.html:37]: - [ ] Following the best practices in using the correct semantic tags, the Work me section should be inside a `section tag` and not a `div`, I'll suggest you update this :+1: :waffle:

File: index.html [hunk 2]
```diff
@@ -34,15 +34,7 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
...
       <img class="Snapshoot2" src="image/SnapshootD3.png" alt="">
```
Comment (V-Blaze) [inline index.html:38]: - [ ] Your images are missing an `alt attribute`. :thinking: Currently you left them empty, following the best practices, all images are meant to have an alt tag. **The purpose of alt text is to improve accessibility by describing what an image is showing to visitors who do not have the ability to see them. However, it also helps search engine crawlers and so improves SEO.** :smile: :+1:

File: index.html [hunk 3]
```diff
@@ -149,42 +141,24 @@ <h2 class="Tonic"> Multi-Post <br>Stories</h2>
...
-   <p class="Text" >Hello I’m a software developer! I can  <br> help you build a product , feature or  <br> website Look through some of my <br> work and experience! If you like what <br> you see and have a project you need <br> coded, don’t hestiate to contact me.</p>
-
-    <a href="#" class="Connect1">LET’S CONNECT</a>
+ <div class="Top-Sec">
+    <h2 class="About" >About Myself </h2>
+     <p class="Text" >Hello I’m a software developer! I can  <br> help you build a product , feature or  <br> website Look through some of my <br> work and experience! If you like what <br> you see and have a project you need <br> coded, don’t hestiate to contact me.</p>
+     <a href="#" class="Connect1">LET’S CONNECT</a>
 
 <ul class="Social1">
   <li><a href="#"><img src="image/Vector5.png" alt=""> </li>
```
Comment (V-Blaze) [inline index.html:164]: - [ ] Your images are missing an `alt attribute`. :thinking: Currently you left them empty, following the best practices, all images are meant to have an alt tag. **The purpose of alt text is to improve accessibility by describing what an image is showing to visitors who do not have the ability to see them. However, it also helps search engine crawlers and so improves SEO.** :smile: :+1:  I'll suggest you go through your code and add texts in the alt attribute for all the images

File: index.html [hunk 4]
```diff
@@ -149,42 +141,24 @@ <h2 class="Tonic"> Multi-Post <br>Stories</h2>
...
+   <div class="Languages5">
       <button type="submit" value="Submit" class="lan51">html</button>
       <button type="submit" value="Submit" class="lan52">css</button>
       <button type="submit" value="Submit" class="lan53">javascript</button>
   </div>
 
-
   <div class="See-Project5">
     <button type="submit" value="Submit" class="btn3">See Project</button>
   </div>
```
Comment (V-Blaze) [inline index.html:153]: - [ ] Your `See Project` and `Get resume` button doesn't have the button state styling implemented, you are meant to implement the `hover and pressed` state for your button according to the Figma design.  I'll suggest you implement this for all the right buttons in your project.

- - FIGMA DESIGN BUTTON STATES:
Screenshot from 2023-06-02 12-18-10

File: index.html [hunk 5]
```diff
@@ -34,15 +34,7 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
...
-
-
-
-   
-
-
-    <div class="Container2">
+   <div class="Container2">
       <img class="Snapshoot2" src="image/SnapshootD3.png" alt="">
       <h2 class="Tonic">  Tonic</h2>
```
Comment (V-Blaze) [inline index.html:39]: - [ ] Your project title seems to have different sizes, some are small and some are bigger, I'll suggest you go to the Figma design and get the correct size and implement it across all the sizes. :+1: 

- -  This is small:
Screenshot from 2023-06-02 12-25-13

- - And this is bigger:
Screenshot from 2023-06-02 12-24-56

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Tshepo45 , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct GitHub flow✔
- Readme looks great👋 ✔

 ## Required Changes ♻️
- [ ] Your code seems not to be organized and well structured. there are too many spaces between divs, and the opening and closing tags of HTML tags are not aligned properly. I'll suggest you get a code formatted expansion like `Prettier`, A clean code base makes the code easy to understand and work with. :smile: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -23,212 +23,154 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
...
-
-
-
-   
-
+  </div>
 
-    <div class="Container2">
-      <img class="Snapshoot2" src="image/SnapshootD3.png" alt="">
+   <section class="Container2">
```
Comment (franclobo) [inline index.html:35]: - **[OPTIONAL]** In the work section all the cards have a margin, please add it to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Tshepo45, 

# ♻️ Required Changes ♻️

soClose

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @Tshepo45 , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**